src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-field.html

```diff
@@ -1,51 +1,78 @@
-
-<!-- Create a search bar form with a text input and a dropdown selector for the search index -->
-<form
-  class="bd-search d-flex align-items-center"
+{% if theme_use_meilisearch %}
+  {% set API_KEY = theme_use_meilisearch.api_key %}
+  <!-- Include the MeiliSearch CSS stylesheet only if theme present -->
+  <link href="{{ pathto('_static/css/meilisearch.css', 1) }}" rel="stylesheet" />
+  <!-- Create a search bar form with a text input and a dropdown selector for the search index -->
+  <form class="bd-search d-flex align-items-center"
   action="{{ pathto('search') }}"
-  method="get"
->
-  <i class="fa fa-search" aria-hidden="true" id="search-icon"></i>
-  <input
-    type="search"
-    placeholder="Search"
+  method="get">
+  <i class="fa-solid fa-magnifying-glass"></i>
+  <input type="search"
     class="form-control"
-    id="search-bar-input"
-    autofocus
-  />
-  <span class="search-button__kbd-shortcut"
-    ><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span
-  >
-  <style>
-    .bd-search .search-button__kbd-shortcut {
-      right: 33%;
-    }
-  </style>
-  {% if theme_use_meilisearch.index_uids %}
-  <select id="indexUidSelector" class="index-select">
+    name="q"
+    placeholder="{{ theme_search_bar_text }}"
+    aria-label="{{ theme_search_bar_text }}"
+    autocomplete="off"
+    autocorrect="off"
+    autocapitalize="off"
+    spellcheck="false"/>
+  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
+  <!-- Conditionally display the dropdown selector if there are multiple indices -->
+  {% if theme_use_meilisearch.index_uids | length > 1 %}
+    <select id="indexUidSelector" class="index-select">
     {% for index_uid, index_name in theme_use_meilisearch.index_uids.items() %}
-    <option value="{{ index_uid }}">{{ index_name }}</option>
+      <option value="{{ index_uid }}">{{ index_name }}</option>
     {% endfor %}
-  </select>
+    </select>
+    <style>
+      .form-control:focus,.form-control:focus-visible,.form-control {
+        width: 490px;
+      }
+    </style>
   {% endif %}
   <!-- Include the MeiliSearch JavaScript library for the search bar -->
   <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js"></script>
-
-  {% if not theme_use_meilisearch.host %} {% set HOST_URL =
-  "https://backend.search.pyansys.com/" %} {% else %} {% set HOST_URL =
-  theme_use_meilisearch.host %} {% endif %}
+  {% if not theme_use_meilisearch.host %}
+    {% set HOST_URL = "https://backend.search.pyansys.com/" %}
+  {% else %}
+    {% set HOST_URL = theme_use_meilisearch.host %}
+  {% endif %}
   <script>
-    var HOST_URL = "{{ HOST_URL }}";
-    var API_KEY = "{{ API_KEY }}";
-    var indexUid = "{{ theme_use_meilisearch.index_uids.items()|first|first }}";
-  </script>
+  var HOST_URL = "{{ HOST_URL }}";
+  var API_KEY = "{{ API_KEY }}";
+  var indexUid = "{{ theme_use_meilisearch.index_uids.items()|first|first }}";
```

Comment:
Why are we applying the `first` filter twice? Mind you adding a comment explaining it?

Reply:
This code iterates over the `index_uids` and `index_names` dictionaries to access the first child dictionary, and then retrieves the first key from that child dictionary to obtain the `index_uid`. i addede this inline comment.